setup.py

```diff
@@ -62,5 +62,6 @@
         "elastic-apm": ["elastic-apm<7.0.0,>=6.15.1"],
         "fastapi": ["fastapi<1.0.0,>=0.100"],
         "rich": ["rich"],
+        "dev": ["graphviz"],
```

Comment:
I've modified this to use `dev`. This generic name could be used for different requirements as `graphviz` in this nice cli feature.

Reply:
Yes, much better